draft-fdb-rats-psa-endorsements.md

```diff
@@ -40,39 +40,44 @@ author:
 
 normative:
   PSA-TOKEN: I-D.tschofenig-rats-psa-token
-
   CoRIM: I-D.ietf-rats-corim
 
 informative:
   RATS-ARCH: RFC9334
-
+  TEEP: I-D.ietf-teep-architecture
   PSA-CERTIFIED:
    target: https://www.psacertified.org
    title: PSA Certified
    date: 2021
 
+entity:
+  SELF: "RFCthis"
+
 --- abstract
 
-PSA Endorsements include reference values, cryptographic key material and
-certification status information that a Verifier needs in order to appraise
-attestation Evidence produced by a PSA device.  This memo defines such PSA
-Endorsements as a profile of the CoRIM data model.
+PSA Endorsements include reference values, endorsed values, cryptographic key
+material and certification status information that a Verifier needs in order to
+appraise attestation Evidence produced by a PSA device.  This memo defines such
```

Comment:
I think, we agreed that Certification Status bakes into "Verifier Added Claims" which are strictly not required to appraise attestation Evidence. Hence we should set the expectations and "bar" accordingly for certification status, right from the start.

Reply:
will use "may need" instead of "needs"